base/src/test/java/com/mindera/skeletoid/logs/appenders/LogFileAppenderUnitTest.kt

```diff
@@ -252,6 +253,7 @@ public class LogFileAppenderUnitTest {
     }
 
     @Test
+    @Ignore
```

Comment:
I'm totally for it, this is a very unstable test I'm afraid...

Reply:
I was hoping it would pass with 5 seconds.
Should we try more seconds ?

Reply:
We can always try!